to.etc.domui/src/main/java/to/etc/domui/dom/html/RadioGroup.java

```diff
@@ -212,6 +212,10 @@ static public <T extends Enum<T>> RadioGroup<T> createEnumRadioGroup(Class<T> cl
 	}
 
 	static public <T extends Enum<T>> RadioGroup<T> createEnumRadioGroup(Class<T> clz, @Nullable IRenderInto<ValueLabelPair<T>> valueRenderer, T... exceptions) {
+		return createEnumRadioGroup(clz, true, valueRenderer, exceptions);
+	}
+
+	static public <T extends Enum<T>> RadioGroup<T> createEnumRadioGroup(Class<T> clz, boolean sorted, @Nullable IRenderInto<ValueLabelPair<T>> valueRenderer, T... exceptions) {
```

Comment:
I do not like boolean parameters in public interface methods. Can we:
- make this method private
- add another public method createEnumRadioGroupUnsorted() instead?

Reply:
ok, fixed